docs/submit-opentargets-batch.md

```diff
@@ -81,27 +95,37 @@ cd ${CODE_ROOT} && ${BSUB_CMDLINE} \
   -j clinvar-xml-parser/src/main/java
 ```
 
-A new Java package will be generated in the directory `clinvar-xml-parser/src/main/java/uk/ac/ebi/eva/clinvar/model`. With each schema version change, test data must be updated as well. See details in [Build instructions](build.md#regenerating-test-data). Create a pull request to merge this code into the main repository. It must contain both the updated schema version as well as the new test data. After a schema update, you'll also need to rebuild Java parser (see [Build instructions](build.md#building-java-clinvar-parser)).
+A new Java package will be generated in the directory `clinvar-xml-parser/src/main/java/uk/ac/ebi/eva/clinvar/model`. You will need to rebuild the parser; see [Build instructions](build.md#building-java-clinvar-parser).
+
+Update the test data. All the test does (for the moment) is checking that parsing 10 records from the XML will (1) not crash and (2) provide 10 records after parsing. So to regenerate test data, we just have to extract any 10 records (can just be the first 10 records) from the ClinVar XML file:
+```bash
+zcat ${BATCH_ROOT}/clinvar/ClinVarFullRelease_${CLINVAR_RELEASE}.xml.gz \
+  | awk 'BEGIN {RS="</ClinVarSet>\n\n"; ORS=RS} {print} NR==10 {exit}' \
+  > ${CODE_ROOT}/clinvar-xml-parser/src/test/resources/ClinvarExample.xml
+echo "</ReleaseSet>" >> ${CODE_ROOT}/clinvar-xml-parser/src/test/resources/ClinvarExample.xml
+gzip -c \
+  <${CODE_ROOT}/clinvar-xml-parser/src/test/resources/ClinvarExample.xml \
+  >${CODE_ROOT}/clinvar-xml-parser/src/test/resources/ClinvarExample.xml.gz
```

Comment:
Yes, that's much nicer, thank you for the suggestion! The only adjustment I had to make to this command is to add `tee`, because the current test suite design requires both of those files (the compressed and the uncompressed one). Addressed in https://github.com/EBIvariation/eva-cttv-pipeline/pull/87/commits/5fd796feef7a4ce6c2f61ae181d71013f4150bc3, and now it looks like this:

```bash
zcat ${BATCH_ROOT}/clinvar/ClinVarFullRelease_${CLINVAR_RELEASE}.xml.gz \
  | awk 'BEGIN {RS="</ClinVarSet>\n\n"; ORS=RS} {print} NR==10 {exit} END {print "</ReleaseSet>"}' \
  | tee ${CODE_ROOT}/clinvar-xml-parser/src/test/resources/ClinvarExample.xml \
  | gzip -c >${CODE_ROOT}/clinvar-xml-parser/src/test/resources/ClinvarExample.xml.gz
```